lib/portlayer/exec/container.go

```diff
@@ -377,15 +382,23 @@ func (c *Container) Signal(ctx context.Context, num int64) error {
 	return c.startGuestProgram(ctx, "kill", fmt.Sprintf("%d", num))
 }
 
-func (c *Container) LogReader(ctx context.Context) (io.ReadCloser, error) {
+func (c *Container) stopped() {
```

Comment:
Looks like this only deals with the log reader.  Why not call it stopLogReaders() or something?  Not a big deal.